### PR TITLE
selfRegister cleanup fix

### PR DESCRIPTION
**Pull request by StanislavStefanov**
**Description**

Changes proposed in this pull request:
- fix cleanup self register delete request URL building

**Pull Request status**

- [X] Implementation
- [X] Unit tests
- [X] Integration tests
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

File: components/director/internal/domain/runtime/resolver.go [hunk 1]
```diff
@@ -301,14 +305,14 @@ func (r *Resolver) DeleteRuntime(ctx context.Context, id string) (*graphql.Runti
 		return nil, err
 	}
 
-	selfRegLabel, err := r.runtimeService.GetLabel(ctx, runtime.ID, r.selfRegManager.GetSelfRegDistinguishingLabelKey())
+	_, err = r.runtimeService.GetLabel(ctx, runtime.ID, r.selfRegManager.GetSelfRegDistinguishingLabelKey())
```
[the remark this answers is not in the document]
Comment (StanislavStefanov) [reply components/director/internal/domain/runtime/resolver.go:308]: We still need the method as we need to know whether such label exists. If it exists we should provide runtime id for the cleanup, but if it does not exist we should pass empty value. We determine the label existance based on the error, but the actual label value is not important.

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**